src/core/sys/windows/winreg.d

```diff
@@ -97,58 +97,58 @@ static if (_WIN32_WINNT >= 0x600) {
 }
 
 extern (Windows) nothrow @nogc {
-    LONG RegCloseKey(in HKEY);
+    LONG RegCloseKey(const HKEY);
```

Comment:
Why `const` here instead of `scope const`?

Reply:
Because `HKEY` is a value type.

Reply:
Actually, I think `HKEY` may be `const(void*)`.  It's defined here here:  https://github.com/dlang/druntime/blob/8fd52019826259dc92ab712f4b37a3f0ea4d8265/src/core/sys/windows/windef.d#L94

And `DECLARE_HANDLE` is defined here:  https://github.com/dlang/druntime/blob/8fd52019826259dc92ab712f4b37a3f0ea4d8265/src/core/sys/windows/basetsd.d#L41-L50

I'll update it to `const scope`.

Reply:
I think there was a whole pr that you wrote with HKEY as const instead of const scope.

Reply:
That was this PR.

Reply:
It seems it was, for some reason I thought I had merged it already.